configs/configuration.yaml

```diff
@@ -1,5 +1,5 @@
 config_name: cytopipe_defaults
 config_paths:
-  single_cell: "config/analysis_confings/single_cell_config.yaml"
-  annotate: "config/analysis_confings/annotate_config.yaml"
-  normalize: "config/analysis_confings/normalize_config.yaml"
+  single_cell: "configs/analysis_configs/single_cell_configs.yaml"
```

Comment:
i was confused for a second about `single_cell` config pertaining to the `aggregation` step.

Specifically this line:

```python
aggregate_configs = yaml.safe_load(yaml_contents["single_cell_config"]["params"]
```

I did not expect the variable `aggregate_configs` to be load the `single_cell_config`. Maybe change the variable name to match the config name?

Reply:
(i.e. the actual change happens in `aggregate_cells.py`)?

Reply:
Now that you have mentioned it.... Yeah it is a bit confusing.

The `single_cell_config` actually refers to the `SingleCell` class in `pycytominer`. The idea was to let the user know which Class or function configuration they are looking at. 

We can change the name of the config using the name of the rule. 

For example:
Since the name of the first rule is `aggregate` we can  switch the `single_cell_config.yaml` name  to `aggregate_config.yaml`

Reply:
Sounds good!